workbench

```diff
@@ -17,7 +17,7 @@ import requests_cache
 from progress_bar import InitBar
 from workbench_utils import *
 import workbench_fields
-
+from WorkbenchConfig import WorkbenchConfig
 
```

Comment:
`pycodestyle --show-source --show-pep8 --ignore=E402,W504 --max-line-length=200 workbench` says you need an extra blank line here. Otherwise, works as advertised.